packages/wpcom-checkout/src/partner-coupon.ts

```diff
@@ -0,0 +1,16 @@
+import { getSublabel } from './checkout-labels';
+import type { ResponseCartProduct } from '@automattic/shopping-cart';
+
+export function getPartnerCoupon( {
```

Comment:
Maybe we should add a comment here that similar functionality exists in `client/jetpack-connect/controller.js` => `partnerCouponRedirects()` (and vice versa) ? 
The `JETPACK_COUPON_PARTNERS` constant it uses could be a singular list containing all supported partners, so we only have to maintain them in one place - it just needs to be an object that's returned instead of a partner string 😄 
But IMO, a comment should be OK for now, so we can get it live 👍